ocs_ci/helpers/helpers.py

```diff
@@ -4576,6 +4564,29 @@ def retrieve_cli_binary(cli_type="mcg"):
     ), f"The {cli_type} CLI binary does not have execution permissions"
 
 
+def download_pull_secret():
```

Comment:
This can be added to ocs_ci/deployment/ocp.py. There is already a function for getting content of the pull secret file (https://github.com/red-hat-storage/ocs-ci/blob/503f071e820544889b09b9044ef507c74a2676f2/ocs_ci/deployment/ocp.py#L64)

Reply:
due to the hierarchy, I mostly use it from other functions or helper objects like HypershiftBase I moved it to a file ocs-ci/ocs_ci/deployment/ocp.py but left it be a function and not a method of OCPDeployment